documentation/source/release_notes.rst

```diff
@@ -4,6 +4,27 @@ Release Notes
 
 All releases are available from the `GitHub Releases Page <https://github.com/cocotb/cocotb/releases>`_.
 
+cocotb 1.3
+==========
+
+Released on 31 December 2019
+
+New features
+------------
+* :func:`cocotb.triggers.with_timeout`, for a shorthand for waiting for a trigger with a timeout.
+* New ``timeout_time`` and ``timeout_unit`` arguments to :func:`cocotb.test`, for adding test timeouts.
+* The ``expect_error`` argument to :func:`cocotb.test` now accepts a specific exception type.
+* :envvar:`COCOTB_RESULTS_FILE`, to allow configration of the xunit XML output filename.
```

Comment:
A ``make changes`` in ``documentation/`` shows that ``COCOTB_HDL_TIMEUNIT`` and ``COCOTB_HDL_TIMEPRECISION`` have also been added, and says that ``raise_error`` and ``create_error`` are deprecated in favor of ``TestError``.
Also, Verilator support!

Reply:
Mind adding the bullet points you'd like to see added in a comment?

Reply:
Maybe like this:
```
* New makefile variables :make:var:`COCOTB_HDL_TIMEUNIT`/:make:var:`COCOTB_HDL_TIMEPRECISION` for setting the default time unit/precision that should be assumed for simulation when not specified by modules in the design.
```
```
* Support for the :ref:`Verilator` simulator (in version 4.020 and above.)
```

Reply:
Probably worth a new deprecations subheading